src/components/faq/Faq.js

```diff
@@ -14,12 +16,14 @@ function FAQ() {
                 </h2>
                 </div>
                 <div className="faq__items">
-                    {FAQ_ITEMS.map(item => (
-                        <div key={item.title} className="faq__item br-xs">
-                            <h3 className="faq__subtitle">{item.title}</h3>
-                            <Icon className="faq__icon" name="plus" />
-                            <p className="faq__paragraph">{item.answer}</p>
-                    </div>
+                    {FAQ_ITEMS.map((item, i) => (
+                        <div key={i} className={`faq__item br-xs ${active === i}`} onClick={() => {
+                            setActiveBlock(active === i ? "" : i)
```

Comment:
and set `null` instead of `""` here.

Reply:
`i` - its a number. active by default is string. `active === i` will always return `false`.